tests/migration_test.go

```diff
@@ -1142,7 +1143,8 @@ var _ = Describe("[Serial][rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][leve
 				tests.CreateNFSPvAndPvc(pvName, util.NamespaceTestDefault, "5Gi", nfsIP, os)
 
 				cfg := getCurrentKv()
-				cfg.MigrationConfiguration.BandwidthPerMigration = resource.NewMilliQuantity(1, resource.BinarySI)
+				migrationBandwidth = resource.MustParse("40Mi")
```

Comment:
`migrationBandwidth` is only here. You don't have to define it outside of that context.

Reply:
Thanks for catching that. Probably a leftover